app/Libraries/OsuAuthorize.php

```diff
@@ -972,6 +972,17 @@ public function checkCommentVote(?User $user, Comment $comment) : string
         return 'ok';
     }
 
+    public function checkCommentPin(?User $user, Comment $comment) : string
+    {
+        $this->ensureLoggedIn($user);
+
+        if ($user->isAdmin()) {
+            return 'ok';
+        }
```

Comment:
seems like this isn't needed as ``doCheckUser()`` always returns ``ok`` if user is admin